web/src/lib/components/shared-components/user-avatar.svelte

```diff
@@ -13,6 +13,7 @@
     email: string;
     profileImagePath: string;
     avatarColor: UserAvatarColor;
+    updatedAt: string;
```

Comment:
No I mean like send it from the server in the api response.

Reply:
Sure. But can this information be used to know when a user update his private information (like email, password) ?

Reply:
You think exposing updatedAt would be bad because it leaks that other information? 

If so, let's add a new column like profileChangedAt and set it when the profile picture changes.

Reply:
> If so, let's add a new column like profileChangedAt and set it when the profile picture changes.

Yes, IMO that's the best solution. I'll add it next week